tests/monitoring/monitoring.go

```diff
@@ -190,6 +164,57 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 
 })
 
+func disableVirtHandler(virtClient kubecli.KubevirtClient, originalKv *v1.KubeVirt) error {
+	customizedComponents := v1.CustomizeComponents{
+		Patches: []v1.CustomizeComponentsPatch{
+			{
+				ResourceName: virtHandler.deploymentName,
+				ResourceType: "DaemonSet",
+				Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
+				Type:         v1.StrategicMergePatchType,
+			},
+		},
+	}
+
+	patchBytes, err := patch.New(patch.WithAdd("/spec/customizeComponents", customizedComponents)).GeneratePayload()
+	if err != nil {
+		return err
+	}
+
+	_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+	return err
+}
+
+func restoreVirtHandler(virtClient kubecli.KubevirtClient, originalKv *v1.KubeVirt) error {
+	patchBytes, err := patch.New(patch.WithRemove("/spec/customizeComponents")).GeneratePayload()
+	if err != nil {
+		return err
+	}
+
+	_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+	return err
+}
+
+func waitForVirtHandlerNodeSelector(virtClient kubecli.KubevirtClient, namespace, expectedValue string, timeout, polling time.Duration) {
+	Eventually(func() (string, error) {
```

Comment:
`EventuallyWithOffset`

Reply:
fixed 

*it was Eventually before this pr,  but make sense to fix it here. thanks.